extensions/wikia/AdEngine/js/DOMElementTweaker.js

```diff
@@ -0,0 +1,64 @@
+/*global define*/
+define('ext.wikia.adEngine.domElementTweaker', [
```

Comment:
We have https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/utils/domCalculator.js.
Does it make sense to keep those files in the same dir?